testsuite/forge-cli/src/main.rs

```diff
@@ -931,6 +939,20 @@ fn changing_working_quorum_test(
             helm_values["validator"]["config"]["consensus"]
                 ["round_timeout_backoff_exponent_base"] = 1.0.into();
             helm_values["validator"]["config"]["consensus"]["quorum_store_poll_count"] = 1.into();
+
+            // Override the syncing mode of all nodes to use transaction output syncing.
+            // TODO(joshlind): remove me once we move back to output syncing by default.
```

Comment:
maybe we should leave this permanently?

Reply:
Yeah, we could do that... although, we're going to have a new default mode that does both (execute or apply), whichever is cheaper, so we'll probably want to test that? I guess we can have tests for all relevant modes 😄

Reply:
+1 tests for all relevant modes sounds good, along with the new default!